webplugin/template/mck-sidebox.html

```diff
@@ -304,7 +304,9 @@ <h3>No more messages!</h3>
 				</div>
 
 				<div class="mck-conversation-comment n-vis" id="csat-3"></div>
-
+				<div id = "mck-waiting-queue"  class="n-vis">
+					<span id="mck-waiting-queue-1"></span> <span id="mck-waiting-queue-2" ></span><span id="mck-waiting-queue-3" ></span>
+				</div>
```

Comment:
Please format the code, if it's formatted then resolve the comment.

Reply:
Plus give a proper name to the ids by the name you have given I am not able to understand anything.